libindy/src/services/wallet/mod.rs

```diff
@@ -28,7 +28,7 @@ mod storage;
 mod encryption;
 mod query_encryption;
 mod iterator;
-mod language;
+pub mod language;
```

Comment:
We rather need to think about moving the query language out of the wallet service to the proper place (utils may be).
I believe making it public is acceptable for the current, moment. We will change it later. 
Please, add a "TODO"  note.

Reply:
I added all the successful corresponding tests and added a `TODO`.